bin/autoless

```diff
@@ -14,6 +14,7 @@ program
   .option('--interval <ms>', 'How often files are checked for changes', 100)
   .option('--no-watch', "Compile what needs to be compiled and exit")
   .option('--source-map', "Generate source map files next to css files")
+  .option('--auto-prefix <browsers>', "String containing the list of browsers, separated by comma")
```

Comment:
These strings are [Browserslist](https://github.com/ai/browserslist) queries. Something like 'last 2 versions', for example.